pkgs/applications/misc/tessen/default.nix

```
@@ -0,0 +1,52 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, libnotify
+, makeWrapper
+, pass
+, wl-clipboard
+, wtype
+, rofi
+, backend ? rofi
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tessen";
+  version = "2.1.2";
+
+  src = fetchFromGitHub {
+    owner = "ayushnix";
+    repo = "tessen";
+    rev = "v${version}";
+    sha256 = "sha256-MkZM/Au0nC2DPkgGDH5XuSJcGfwOtcjFHvMQhKfqSgY=";
+  };
+
+  dontBuild = true;
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -a tessen $out/bin/tessen
+  '';
+
+  wrapperPath = lib.makeBinPath [
+    backend
+    libnotify
+    pass
+    wl-clipboard
+    wtype
+  ];
+
+  fixupPhase = ''
+    patchShebangs $out/bin
+    wrapProgram $out/bin/tessen --prefix PATH : ${wrapperPath}
+  '';
```

The `wrapperPath` shouldn't be a part of the derivation set. Try this instead:
```suggestion
  fixupPhase = let
    wrapperPath = lib.makeBinPath [
      backend
      libnotify
      pass
      wl-clipboard
      wtype
    ];
  in ''
    patchShebangs $out/bin
    wrapProgram $out/bin/tessen --prefix PATH : ${wrapperPath}
  '';
```